src/go/rpk/pkg/cli/cmd/topic/add_partitions.go

```diff
@@ -25,12 +25,20 @@ import (
 
 func newAddPartitionsCommand(fs afero.Fs) *cobra.Command {
 	var num int
+	var force bool
 	cmd := &cobra.Command{
 		Use:   "add-partitions [TOPICS...] --num [#]",
 		Short: "Add partitions to existing topics",
 		Args:  cobra.MinimumNArgs(1),
 		Long:  `Add partitions to existing topics.`,
 		Run: func(cmd *cobra.Command, topics []string) {
+			if !force {
+				for _, t := range topics {
+					if t == "__consumer_offsets" {
+						out.Exit("It's generally a bad idea to change the partition count in __consumer_offsets. Exiting.\nAdd the '-f' flag if you really want to change it.")
```

Comment:
We have --no-confirm on a few commands, but I think --force makes more sense here. I need to to think on that a bit.

We probably want to change the error message here to something like:
```
Unable to change __consumer_offsets without the --force flag.
```
(i.e., to drop the line about "generally a bad idea")